src/modules/syslink/syslink_bridge.cpp

```diff
@@ -111,21 +111,45 @@ SyslinkBridge::read(struct file *filp, char *buffer, size_t buflen)
 ssize_t
 SyslinkBridge::write(struct file *filp, const char *buffer, size_t buflen)
 {
-	crtp_message_t msg;
 
-	// Queue and send next time we get a RAW radio packet
-	int remaining = buflen;
+	static bool init = 1;
+
+	if (init) {
+		_msg_to_send.header = 0;
```

Comment:
Please move this block to the constructor (https://github.com/barzanisar/Firmware/blob/60e59ff357c1b111bbe261cb6671c7d8bd8bdd24/src/modules/syslink/syslink_bridge.cpp#L50), then you don't need the `init` flag here.